contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle/common.py

```diff
@@ -259,7 +260,7 @@ def flatten_lines(*line_or_line_list):
 
   def __init__(self, code, severity, filename, message, line_range=None, lines=None):
     if severity not in self.SEVERITY:
-      raise ValueError('Severity should be one of {}'.format(' '.join(self.SEVERITY.values())))
+      raise ValueError('Severity should be one of {}'.format(' '.join(list(self.SEVERITY.values()))))
```

Comment:
Likewise I think you don't need list(). The str.format() will call the values when it needs them, avoiding the intermediate call to list().

Reply:
You do otherwise it takes str() of generator which isn't what we want